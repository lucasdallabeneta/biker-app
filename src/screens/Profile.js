/* eslint-disable react-native/no-inline-styles */
/* eslint-disable prettier/prettier */
//import AsyncStorage from '@react-native-community/async-storage';

import React from 'react';
import {
  View,
  SafeAreaView,
  Text,
  Image,
  Button,
  TextInput,
  TouchableOpacity,
  Alert,
  KeyboardAvoidingView,
  Platform,
} from 'react-native';
//import api from '../services/api';
import Estilos from '../Estilos';
// import imagem1 from '../imgs/computer.png';
// import imagem2 from '../imgs/emoji.png';
// import imagem3 from '../imgs/family.png';
// import imagem4 from '../imgs/horse.png';
// import imagem5 from '../imgs/people.png';
// import imagem6 from '../imgs/taco.png';

export default class Main extends React.Component {

  constructor(props) {
    super(props);
    this.state = {
      parametros:  this.props.navigation.getParam('nomeTeste', 'erro'),
    };
  }

  // static navigationOptions = {
  //   title: 'Parceiros',
  // };

  // componentDidMount() {
  //   console.log("NAV: ", this.props.navigation.getParam('nome'));
  // }

  // static navigationOptions = ({ navigation }) => {
  // static navigationOptions = props => {
  //   const { navigation } = props;
  //   return {
  //     //headerTitle: `Usu√°rio: ${navigation.getParam('nomeTeste', 'ERROR')}`,
  //     //headerRight: (<Button title='Purple' onPress={() => navigation.navigate('TelaAgenda')}/>)
  //   };
  // };

  render() {
    return (
      <View style={{ flex:1 }}>
        <View style={[Estilos.containerMain,{ flex : 1 }]}/>
          <View style={[Estilos.containerMain,{ flex : 5 , flexDirection: 'column' }]}>
            <Text>
              teste
            </Text>
          </View>
        <View style={[Estilos.containerMain,{ flex : 1 }]} />
      </View>
    );
  }
}
