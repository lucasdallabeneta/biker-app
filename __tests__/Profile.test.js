import React from 'react';
import Profile from '../src/screens/Profile';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<Profile />).toJSON();
  expect(tree).toMatchSnapshot();
});