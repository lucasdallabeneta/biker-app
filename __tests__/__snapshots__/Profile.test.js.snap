// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "margin": 5,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#f92",
        "borderColor": "black",
        "borderRadius": 100,
        "flexDirection": "row",
        "height": 120,
        "margin": 5,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <Image
        source={
          Object {
            "testUri": "../../../src/imgs/horse.png",
          }
        }
        style={
          Object {
            "resizeMode": "contain",
          }
        }
      />
    </View>
    <View
      style={
        Object {
          "flex": 2,
          "justifyContent": "center",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#FFF",
            "fontSize": 20,
            "fontWeight": "bold",
          }
        }
      >
          bem vindo 
        
         
      </Text>
      <Text
        style={
          Object {
            "color": "#FFF",
            "fontSize": 20,
          }
        }
      >
          funcionario da bikerapp 
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "backgroundColor": "#f92",
        "borderColor": "black",
        "borderRadius": 100,
        "flexDirection": "row",
        "height": 120,
        "margin": 5,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#FFF",
            "fontSize": 20,
          }
        }
      >
          creditos diponiveis: -1 
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "space-between",
        "margin": 5,
      }
    }
  >
    <View
      accessible={true}
      focusable={false}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignSelf": "stretch",
          "backgroundColor": "#f92",
          "borderRadius": 50,
          "fontWeight": "bold",
          "height": 50,
          "justifyContent": "center",
          "marginHorizontal": 50,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#fff",
            "fontWeight": "bold",
            "paddingHorizontal": 30,
          }
        }
      >
        Meus cupons e descontos
      </Text>
    </View>
    <View
      accessible={true}
      focusable={false}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignSelf": "stretch",
          "backgroundColor": "#f92",
          "borderRadius": 50,
          "fontWeight": "bold",
          "height": 50,
          "justifyContent": "center",
          "marginHorizontal": 50,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#fff",
            "fontWeight": "bold",
            "paddingHorizontal": 30,
          }
        }
      >
        Editar Perfil
      </Text>
    </View>
    <View
      accessible={true}
      focusable={false}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignSelf": "stretch",
          "backgroundColor": "#f92",
          "borderRadius": 50,
          "fontWeight": "bold",
          "height": 50,
          "justifyContent": "center",
          "marginHorizontal": 50,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#fff",
            "fontWeight": "bold",
            "paddingHorizontal": 30,
          }
        }
      >
        Configurações do Aplicativo
      </Text>
    </View>
    <View
      accessible={true}
      focusable={false}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignSelf": "stretch",
          "backgroundColor": "#f92",
          "borderRadius": 50,
          "fontWeight": "bold",
          "height": 50,
          "justifyContent": "center",
          "marginHorizontal": 50,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#fff",
            "fontWeight": "bold",
            "paddingHorizontal": 30,
          }
        }
      >
        Ajuda
      </Text>
    </View>
    <View
      accessible={true}
      focusable={false}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignSelf": "stretch",
          "backgroundColor": "#f92",
          "borderRadius": 50,
          "fontWeight": "bold",
          "height": 50,
          "justifyContent": "center",
          "marginHorizontal": 50,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#fff",
            "fontWeight": "bold",
            "paddingHorizontal": 30,
          }
        }
      >
        Feedback
      </Text>
    </View>
    <View
      accessible={true}
      focusable={false}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignSelf": "stretch",
          "backgroundColor": "#f92",
          "borderRadius": 50,
          "fontWeight": "bold",
          "height": 50,
          "justifyContent": "center",
          "marginHorizontal": 50,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#fff",
            "fontWeight": "bold",
            "paddingHorizontal": 30,
          }
        }
      >
        Sobre
      </Text>
    </View>
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignSelf": "stretch",
          "backgroundColor": "#f92",
          "borderRadius": 50,
          "fontWeight": "bold",
          "height": 50,
          "justifyContent": "center",
          "marginHorizontal": 50,
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Object {
            "alignSelf": "stretch",
            "color": "#fff",
            "fontWeight": "bold",
            "paddingHorizontal": 30,
          }
        }
      >
        Sair
      </Text>
    </View>
  </View>
</View>
`;
